free-2.md

```diff
@@ -8,44 +8,103 @@ good way to meet their needs.
 
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
-experience like now? What would their experience be like if you could help 
+experience like now? What would their experience be like if you could help
 them?_
 
+Playlists of songs are currently a simple ordering of specific tracks to play.
+If a music listener wants to hear new music without explicitly saying what
+they want to hear they may turn to discovery services like Spotify or Pandora;
+however, these services lack fine-grained control over what songs you listen to
+usually relying on factors like what you've listened to in the past and whether
+or not you thumbs up a song.
+
+It would be extrmeley nice if one could specify with more paramaeters what they
+want to listen to. For instance, specify a genre and a deviation from that genre,
+the number of songs per artist or band to be played in given amount of time, or
+beats per minute of a song in the playlist.
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A simple thumbs up or thumbs down (or even an invisble predictive algorithm) do
+not offer enough control. With a DSL, one could specify what they want to hear
+with more expressiveness and clarity. This DSL would need to be more natural
+feeling than most programming langugaes to engage the most number of users.
```

Comment:
This seems like it might benefit from being a non-traditional DSL, i.e. a GUI of some kind. Then you can have sliders, buttons, dropdowns, etc and don't need to express yourself with text.